### PR TITLE
some error and typo fixed

### PR DESCRIPTION
**Pull request by hashbone**
some error and typo fixed

### CODE REVIEW

File: op-guide/requirement.md [hunk 1]
```diff
@@ -43,18 +43,20 @@ TiDB 支持部署和运行在 Intel  x86-64 架构的 64位通用硬件服务器
 
 | 组件 | CPU | 内存 | 本地存储 | 网络 | 实例数量(最低要求)|
 | :-- | :-: | :-: | :-: | :-: | :-: |
-| TiDB|  32核+  |  64 GB+  | SAS   最低4块 RAID10， 200 GB+ | 2块+ 千兆网卡| 2|
+| TiDB|  16核+  |  32 GB+  | SAS   最低6块 RAID10， 200 GB+ | 2块+ 千兆网卡| -|
```
Comment (ngaut) [inline op-guide/requirement.md]: 生产环境推荐使用更高的配置

**Comment by queenypingcap** [comment]
@zhengwanbo @sunhao2017 PTAL

**Comment by zhengwanbo** [comment]
no system monitor HW requirements.

**Comment by buggithubs** [comment]
@queenypingcap

**Pull request merged**